management/validation.py

```diff
@@ -179,14 +185,16 @@ def check_qcspec_coverage(dataset_data):
     # now try and add each spec
     spec_report = {}
     for spec in qc_specs.values():
-
+        valid_scf_props = check_scf_props(spec)
```

Comment:
@dotsdl  @pavankum We now validate the scf properties separately to the whole QC specification in this line.